app/components/node-creator/component.js

```diff
@@ -0,0 +1,38 @@
+import Ember from 'ember';
```

Comment:
This component actually has no dom. Just runs some comp on parameters it observes